vite.config.ts

```diff
@@ -2,13 +2,19 @@ import { defineConfig, loadEnv } from "vite";
 import react from "@vitejs/plugin-react";
 import checker from "vite-plugin-checker";
 import tsconfigPaths from "vite-tsconfig-paths";
+import basicSsl from "@vitejs/plugin-basic-ssl";
 
 // https://vitejs.dev/config/
 export default defineConfig(({ command, mode }) => {
   const env = loadEnv(mode, process.cwd(), "");
 
   return {
-    plugins: [react(), tsconfigPaths(), checker({ typescript: true })],
+    plugins: [
+      react(),
+      tsconfigPaths(),
+      checker({ typescript: true }),
+      basicSsl(),
```

Comment:
```suggestion
      basicSsl(),  // used only to enable https on developer workstations
```

Just to avoid confusion, in case a future reader of this code assumes we're using a self-signed cert in production.